platooning_control_ihp/include/platoon_control_ihp_worker.h

```diff
@@ -99,21 +99,36 @@ namespace platoon_control_ihp
         */
         void setCurrentSpeed(double speed);
 
+        /**
+         * \brief UCLA: Put the IHP gap regulation at low level control plug-in
```

Comment:
Description doesn't give any hint at what this method is actually doing.  What does "DesPos" mean?  What quantity is being gotten?  Also, need description of the param and of the return value, including units of measure.